src/Components/CollectionPage/BookInfo.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+import BooksCard from './BooksCard';
+import './CollectionPage.css';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+function BookInfo({ authorId }) {
```

Comment:
I think this is the id of a collection, so I would change the prop name as needed!